### PR TITLE
HIP45 Amendments

### PR DESCRIPTION
**Pull request by lthiery**
This is a draft PR for discussions on fleshing out details of HIP45.

### CODE REVIEW

**Comment by buzzware** [comment]
Some quick ideas 

1) Define a standard for community hotspot votes :
• if regional, only hotspots in the region or wallets with hotspots in the region count
• 1 vote per hotspot, but maximum 100 votes per wallet
• only hotspots and wallets created at least 4 weeks before the committee recommendation or HIP creation count 
2) define a standard for quasi "unanimous". Unanimous outcomes can be used to exit the process and save everybody time, effort and angst eg.
• more than 20 wallets voted
• more than 20 hotspot votes
• the vote is >=90% one way or the other
3) define a standard for lack of interest in an open community vote eg.
  • less than 10 wallets voting
4) After committee recommendation, at least 7 days notice for a town Hall meeting between 7am and 9pm in the affected region's average timezone.
5) At the end of the town Hall meeting, a hotspot vote  is taken in the region. 
6) If near unanimous then the process ends with that outcome , but the author is free to make changes deemed significant enough and resubmit for 1 more vote in 7 days time.
7) If not unanimous and enough interest, then the 4 week discussion period begins.
8) Formal dissent comments must begin with "Formal Dissent" to be counted.

**Comment by lthiery** [comment]
@buzzware I've invited you to be a collaborator on this fork. If you have the time to edit the PR itself with your ideas that'd help things move as fast as possible. You've done a great job here providing some initial suggestions of some "magical numbers" and I realize how much work that takes. However, it'll be easier to work on things if you actually draft it into the HIP format.

File: 0045-lorawan-frequency-plan-selection.md [hunk 1]
```diff
@@ -87,6 +87,9 @@ After steps 2 and 3, before proceeding to the next escalation, DeWi's LoRaWAN
 Committee may withdraw their proposal and reissue a new proposal at anytime.
 This would effectively restart the process at step 1.
 
+## Live Virtual Public Forum
+
```
Comment (mroutsinaus) [inline 0045-lorawan-frequency-plan-selection.md:91]: Is there a way for the DeWi Comittee/Lorawan to notify each hotspot in an effected region when they have made a decision/at the beginning of the 4 week public commentary?

Comment (mroutsinaus) [reply 0045-lorawan-frequency-plan-selection.md:91]: Formal Dissent Shall be determined by writing "I formally dissent" in the comments on the PR.

**Comment by buzzware** [comment]
> @buzzware I've invited you to be a collaborator on this fork. If you have the time to edit the PR itself with your ideas that'd help things move as fast as possible. 

I don't have time sorry. If I have any ideas I'll drop them in here like above.

**Pull request closed without merging**